azkaban-common/src/main/java/azkaban/executor/AbstractExecutorManagerAdapter.java

```diff
@@ -560,6 +557,8 @@ protected void handleCancelFlow(ExecutionReference executionReference,
       ExecutionControllerUtils.finalizeFlow(this.executorLoader, this.alerterHolder,
           executableFlow, "Cancel action has been called but the flow is unreachable.", null,
           finalizingStatus);
+      this.fireEventListeners(Event.create(executableFlow,
```

Comment:
can you add comment to the code changes?

Reply:
Done. Thx!